src/pages/MovieDetails.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react';
+import { useParams, NavLink } from 'react-router-dom';
+import starIcon from '../assets/star.svg';
+
+const IMAGE_BASE_URL = "https://image.tmdb.org/t/p/";
+const POSTER_SIZE = "w342";
+const BACKDROP_SIZE = "w780";
```

Comment:
Great that you created variables for the sizes, which I didn't think of.

Reply:
I agree with Hang, this was a great idea!